main/fdm_pl.po

```diff
@@ -1499,9 +1499,9 @@ msgstr "Brak połączenia z Internetem"
 msgctxt "MainStatusBar|"
 msgid "%n downloads selected."
 msgid_plural "%n downloads selected."
-msgstr[0] ""
-msgstr[1] ""
-msgstr[2] ""
+msgstr[0] "Jedno pobieranie zaznaczone"
```

Comment:
```suggestion
msgstr[0] "%n pobieranie zaznaczone."
```

Reply:
> 

Right, but if this is "one" in Polish we are writting "Jedno /something/", not "1 /something/".
Thanks for dots at the end of strings.